js/index.js

```diff
@@ -1,6 +1,70 @@
 // Iteration 1: Names and Input
 
+const hacker1 = 'Neo'
+
+console.log(`The driver's name is ${hacker1}`)
+
+const hacker2 = 'Trinity'
+
+console.log(`The navigator's name is ${hacker2} `)
+
+
+
 // Iteration 2: Conditionals
 
+if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`)
+}
+else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`)
+}
+
+else if (hacker1.length === hacker2.length) {
+    console.log(`Wow, you both have equally long names, ${hacker1.length} characters!`)
+}
```

Comment:
nesse caso não precisaria ser `else if`, poderia ser um `else` ou nem ter nada, só o `console.log` direto, porque se não for maior, nem menor, com certeza é igual 😉